### ISSUE TITLE
Training on a NEW language

### ISSUE THREAD

**Issue by rudransh2004** [huggingface/parler-tts#25]
Suppose we have to train this TTS model on a language and the tokens of that language are not in the Flan-T5 transformer. So can I simply change the name of the tokenizer in the config.json or do I have to make any code changes also. NOTE  The new tokenizer will not be of FLAN-T5

**Comment by ylacombe** [comment]
Hey @rudransh2004, you can do this but you'd have to retrain the model from scratch!

**Comment by rudransh2004** [comment]
Hey @ylacombe  thank you soo much for your reply. Could you share some reciepe for doing this with some another language without the annotations if we wish to

**Comment by ylacombe** [comment]
Hey @rudransh2004, if you want to avoid using the annotations, you could simply use a description column with empty string "".
Note that the model currently doesn't support passing samples without annotations, but the trick above should work

**Comment by chunping-xt** [comment]
Hi @ylacombe, I'm trying finetuning parler-tts/parler_tts_mini_v0.1 with my language and I noticed the tokenizer of parler-tts/parler_tts_mini_v0.1 and Flan-T5 doesn't support it (has <unk> when decoding). I tried replacing the tokenizer with mT5 because it supports my language, but it gave me an error. If I continue with tokenizer parler_tts_mini_v0.1, training with a small amount of data about 25 hours many speakers, with prompt = transcription, after 50 epochs the result is unstable loss, eval decreases slowly, the audio part is unlike any other language that exists in the world. My Questions:
1. Which model should I start with?
2. Minimum amount of training data? I just need to see that the model works well with my data, I have enough big data for the next training.
3. Is there a solution to tokenzier supporting my language, or continue with the current tokenizer and ignore unknow tokens?
4. I see whisper participating in the training process, do I have to choose a whisper model that works well with my data, for example if I later plan to train on singing data?
